tools/bazar/controllers/ApiController.php

```diff
@@ -173,6 +174,65 @@ public function createSemanticEntry($formId)
         ]);
     }
 
+    /**
+     * @Route("/api/bazar-list-data", methods={"GET"}, options={"acl":{"public"}})
+     */
+    public function getBazarListData()
+    {
+        try {
+            $bazatListService = $this->getService(BazarListService::class);
+            $forms = $bazatListService->getForms($_GET);
+            $entries = $bazatListService->getEntries($_GET, $forms);     
```

Comment:
```suggestion
            list($entries,$filters) = $bazatListCommonsController->getEntriesAndFilters([
                     'query' => $_GET['query'] ?? '',
                    'idtypeannonce' => $_GET['idtypeannonce'],
                    'correspondance' => $_GET['correspondance'] ?? ''
            ], $forms);     
```

je serai d'avis que ce soit la responsabilité de ApiController (controller primaire) de nettoyer `$_GET` et de ne passer à BazarListCommonsController uniquement les valeurs nécessaires.